android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java

```diff
@@ -2368,7 +2382,7 @@ public void reportJSException(String instanceId, String function,
       exception +=   "\n getTemplateInfo==" +instance.getTemplateInfo();//add network header info
       if (METHOD_CREATE_INSTANCE.equals(function) || !instance.isContentMd5Match()) {
         try {
-          if (isJSFrameworkInit() && (reInitCount > 1 && reInitCount < 10) && !instance.isNeedReLoad()) {
+          if (!isSkipFrameworkInit(instanceId) && isJSFrameworkInit() && (reInitCount > 1 && reInitCount < 10) && !instance.isNeedReLoad()) {
```

Comment:
I think this line is wrong, consider changing to 

    ( isSkipFrameworkInit(instanceId) || isJSFrameworkInit() ) && (reInitCount > 1 && reInitCount < 10) && !instance.isNeedReLoad()